### ISSUE TITLE
[1.2.5] amazon-import post-processor 

### ISSUE THREAD

**Issue by lmayorga1980** [hashicorp/packer#6542]
I got around 5 errors with the latest packer when importing an OVA (vmware-iso) exported machine to aws.

```
Post-processor failed: Import task <task-id> failed with error: ResourceNotReady: exceeded wait attempts
```

It worked fine on [1.2.4]

**Comment by SwampDragons** [comment]
Thanks for opening an issue, and sorry for the regression.  This is a duplicate of #6536; try setting the environment variable `AWS_MAX_ATTEMPTS` (See that other issue for some commenters' notes on what timeouts worked for them.)

**Comment by SwampDragons** [comment]
Yeah, I found a bug where some of our custom-written waiters weren't properly applying the environment overrides; this is fixed on master as you saw, and will be released with Packer 1.3.0 which will go out early next week.
